### PR TITLE
Add Dec 2020 release notes

### PR DESCRIPTION
**Pull request by pinin4fjords**

### CODE REVIEW

File: release-notes/sc/_posts/2020-12-17-13.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#### Data statistics
+
+- Ensembl **99** / Ensembl Genomes **46** / WormBase ParaSite **14** gene annotations.   
+- This release contains **181** single cell RNA-Seq studies, consisting of **5,775,820** cells, of which **4,037,874** passed our QC from **15** different species.
```
Comment (anjaf) [inline release-notes/sc/_posts/2020-12-17-13.md]: Species count is 14 on the website.

**Review by anjaf — COMMENTED** [review]
Otherwise looks good.

**Review by pcm32 — APPROVED** [review]
LGTM

**Comment by pinin4fjords** [comment]
Thanks @anjaf @pcm32 !

**Pull request merged**